packages/core/admin/ee/server/bootstrap.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 // eslint-disable-next-line node/no-extraneous-require
```

Comment:
Unrelated but I don't think this comment is needed

Reply:
makes sense :) Changed!